### ISSUE TITLE
nym-vpn-app 0.2.0 on Debian 12 fails to start

### ISSUE THREAD

**Issue by d-mo** [nymtech/nym-vpn-client#1183]
```
$ nym-vpn-app --version
nym-vpn-app 0.2.0
$ nym-vpn-app 
2024-09-18T19:53:43.077184Z  INFO nym_vpn_app: Creating k/v embedded db
2024-09-18T19:53:43.077234Z  INFO new: nym_vpn_app::fs::path: app data dir: /home/user/.local/share/nym-vpn-app
2024-09-18T19:53:43.077244Z  INFO new: nym_vpn_app::db: opening sled db at /home/user/.local/share/nym-vpn-app/db
2024-09-18T19:53:43.104154Z  INFO new: nym_vpn_app::db: sled db recovered
2024-09-18T19:53:43.104191Z  INFO nym_vpn_app::fs::path: app config dir: /home/user/.config/nym-vpn-app
2024-09-18T19:53:43.104474Z  INFO nym_vpn_app: network environment: mainnet
2024-09-18T19:53:43.104561Z  INFO new: nym_vpn_app::grpc::client: using grpc IPC transport: /var/run/nym-vpn.sock
2024-09-18T19:53:43.104568Z  INFO nym_vpn_app: Starting tauri app
src/nv_gbm.c:288: GBM-DRV error (nv_gbm_create_device_native): nv_common_gbm_create_device failed (ret=-1)

2024-09-18T19:53:43.487211Z  INFO nym_vpn_app: app setup
2024-09-18T19:53:43.496464Z  INFO nym_vpn_app: starting vpnd health spy
2024-09-18T19:53:43.496502Z  INFO nym_vpn_app: starting vpn status spy
2024-09-18T19:53:43.496580Z  INFO nym_vpn_app: starting vpn connection updates spy
2024-09-18T19:53:43.497488Z  INFO refresh_vpn_status:update: nym_vpn_app::vpn_status: vpn status → [Disconnected]
src/nv_gbm.c:288: GBM-DRV error (nv_gbm_create_device_native): nv_common_gbm_create_device failed (ret=-1)

KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
Failed to create GBM buffer of size 739x901: Permission denied
```

**Comment by marcdbz** [comment]
Hello,

Thanks for reporting, we'll look into this issue.

**Comment by rachyandco** [comment]
this should be fixed in the latest 0.2.3 version https://nymvpn.com/en/download/linux
